lib/node_modules/@stdlib/string/remove-last/lib/main.js

```diff
@@ -21,20 +21,39 @@
 // MODULES //
 
 var isString = require( '@stdlib/assert/is-string' ).isPrimitive;
+var isPlainObject = require( '@stdlib/assert/is-plain-object' );
+var hasOwnProp = require( '@stdlib/assert/has-own-property' );
+var contains = require( '@stdlib/array/base/assert/contains' ).factory;
 var isNonNegativeInteger = require( '@stdlib/assert/is-nonnegative-integer' ).isPrimitive;
-var prevGraphemeClusterBreak = require( '@stdlib/string/prev-grapheme-cluster-break' );
+var removeLastCodeUnit = require( '@stdlib/string/base/remove-last' );
+var removeLastCodePoint = require( '@stdlib/string/base/remove-last-code-point' );
+var removeLastGraphemeCluster = require( '@stdlib/string/base/remove-last-grapheme-cluster' );
 var format = require( '@stdlib/string/format' );
 
 
+// VARIABLES //
+
+var MODES = [ 'grapheme', 'code_point', 'code_unit' ];
+var FCNS = {
+	'grapheme': removeLastGraphemeCluster,
+	'code_point': removeLastCodePoint,
+	'code_unit': removeLastCodeUnit
+};
+var isMode = contains( MODES );
+
+
 // MAIN //
 
 /**
 * Removes the last character(s) of a string.
 *
 * @param {string} str - input string
 * @param {NonNegativeInteger} [n=1] - number of character to remove
+* @param {Options} [options] - options
```

Comment:
Missing documentation of `options.mode`. E.g., https://github.com/stdlib-js/stdlib/blob/1b163e9c6c6d81533a9418206e349df3bfca3858/lib/node_modules/%40stdlib/string/remove-first/lib/main.js#L53.